drivers/soundwire/stream.c

```diff
@@ -684,8 +684,6 @@ static int sdw_bank_switch(struct sdw_bus *bus, int m_rt_count)
 	if (!wr_msg)
 		return -ENOMEM;
 
-	bus->defer_msg.msg = wr_msg;
```

Comment:
Why is it required? Where will bus->defer_msg.msg be assigned after this change?

Reply:
it's already done in do_transfer_defer(), so that assignment is redundant. 

````c
	defer->msg = msg;
	defer->length = msg->len;
	init_completion(&defer->complete);
````

Better to do all the 'defer' assignments in one place, the current code is all over the place.